stable/fairwinds-insights/values.yaml

```diff
@@ -321,18 +324,18 @@ openApi:
     max: 3
     # -- Scaling metrics
     metrics:
-    - type: Resource
```

Comment:
Wondering if this may affect current deployment 😬

Reply:
Should not, semantically the same, it's just proper formatted now